deploy.py

```diff
@@ -43,11 +43,25 @@ def ensure_folder(name, uid, api):
 
 
 
-def build_dashboard(dashboard_path):
-    return json.loads(subprocess.check_output([
-        'jsonnet', '-J', 'vendor',
-        dashboard_path
-    ]).decode())
+def build_dashboard(dashboard_path, api):
+    cmd = ["jsonnet", "-J", "vendor", dashboard_path]
+
+    # If we need to build a global dashboard across more datasource
+    # pass the list of datasources to be used in the queries
+    if "global" in dashboard_path:
```

Comment:
If I understand this correctly, we're not deploying some dashboards if we don't have more than one datasource - is that right?

Reply:
Yes, we're not deploying the "global" ones.

So the set of rules would be:
- any dashboards that gather info across multiple datasources, should have the `global` keyword in their filenames
- any dashboards stored in files that contain the `global` keyword need the datasources to be passed as [`tla-code`](https://jsonnet.org/ref/language.html#passing-data-to-jsonnet)
- if a file has the `global` keyword, but we're not actually having more than one datasource, then the dashboards in that file don't get deployed at all.

The best place to document this currently is probably the README, right? What do you think @yuvipanda?

Reply:
README seems great! Can we instead put it in a global/ directory instead of relying on the filename?

Reply:
Actually, since deploy.py takes a 'directory' param, can we:

1. Move these to a 'global-dashboards' directory
2. Wherver they are to be deployed, just run deploy.py twice - once for the local dashboards and once for the global dashboards

This reduces any special casing. What do you think?